<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoalCalendar</class>
 <widget class="QWidget" name="GoalCalendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>181</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="month_ComboBox">
        <item>
         <property name="text">
          <string>January</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>February</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>March</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>April</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>May</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>June</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>July</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>August</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>September</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>October</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>November</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>December</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="year_ComboBox">
        <item>
         <property name="text">
          <string>2013</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2014</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2015</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2016</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2017</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2018</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>180</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="widget_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="UI::CalendarMonth" name="calendarMonth" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>53</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_CustomGoal">
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_DayGoal">
        <property name="text">
         <string>Day</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_MonthGoal">
        <property name="text">
         <string>Month</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_WeekGoal">
        <property name="text">
         <string>Week</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_YearGoal">
        <property name="text">
         <string>Year</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>53</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UI::CalendarMonth</class>
   <extends>QWidget</extends>
   <header location="global">calendarmonth.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
